api/updatepb/update.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package update;
+
+option go_package = "api/updatepb;updatepb";
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+message StartUpdateRequest {
+  // Hostname of the PMM Server
+  string hostname = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message StartUpdateResponse {
+  // Logs token to be used for status.
+  string logs_token = 1;
+}
+
+message UpdateStatusRequest {
+  // Token as returned by StartUpdateResponse.
+  string logs_token = 1;
+  // Offset where to start reading logs.
+  uint32 offset = 2;
+}
+
+message UpdateStatusResponse {
+  // Individual log lines.
+  repeated string lines = 1;
```

Comment:
Should it be called `lines` or just `logs` to be more explicit? It'd echo the `logs_token` mentioned above.

Reply:
I opted for keeping this in sync with the legacy update API so there are minimal changes to the UI.